### ISSUE TITLE
ImportExport NG not working for any type of email export

### ISSUE THREAD

**Issue by Foxfan100** [thundernest/import-export-tools-ng#113]
This was working OK a while ago, but right-click on emails and select an export option produces no result at all. I have checked all folders and reinstalled from both direct add-on and from the downloaded xpi file, but no change.
Thunderbird is 68.8.0 (64-bit) on W10 Pro, 1909 64-bit. ImportExport NG is 4.1.0
![tbird impex](https://user-images.githubusercontent.com/22635836/82435354-16646a80-9a8c-11ea-8b9f-b0f6589767b0.jpg)

This is about all I can see in Debug. If you need more, please advise

**Comment by cleidigh** [comment]
@Foxfan100 
The reference error is very high level and odd.  I will have to try and reproduce on 64bit.
I mostly work on the 32-bit version.
Do you have this problem . after the update to 4.1.0?
any other extensions involved?
@cleidigh

**Comment by cleidigh** [comment]
@Foxfan100 
I am seeing this on a couple extensions.  Something changed with Thunderbird, I'm trying to figure it out.

**Comment by cleidigh** [comment]
@Foxfan100 
one thing you could try start Thunderbird with a command line option:
-purgecaches
This makes sure old JavaScript files are not used.
so far I have not reproduced on Windows I just tried x64 as well

**Comment by pichrocole** [comment]
Hello,
I confirm this bug. 
Impossible to import EML files (nothing happens) with 
Thunderbird 68.8.0 (64-bit) and Thunderbird 68.8.1 on Linux Mint 18.3, ImportExport NG being 4.1.0

Removing version 4.1.0  and reinstalling formal version 4.0.3 solved the problem, everything working again as it should.

Regards

**Comment by Foxfan100** [comment]
I have tried running the 4.1.0 version in several different ways, on W10 Pro 64-bit, and it will not do anything at all; either importing or, for me, exporting emails in various formats. The purgecaches switch has no effect, and I have also tried clearing all Tbird caches via Tools.

I have reinstalled 4.0.4, which works perfectly. For some reason, each of the two versions appear in different places in the right-click context dropdown; older towards the top and newer towards the bottom.

**Comment by cleidigh** [comment]
@Foxfan100 
@picrochole 
sorry for the continued problems!  .I have investigated and only seen what appeared to be similar on OS X.  I will do another round of testing, in the meantime a couple more questions:
- if I serve you are using other extensions have you tried disabling?
- Have you tried with a test profile
@picrochole 
- have you captured any similar debug output ?
- did you also observe the change in menu position?

I think that last point on the menus may be a good clue.
if I cannot reproduce, I will probably have to make test versions With the debug increased

**Comment by picrochole** [comment]
@cleidigh
> have you captured any similar debug output ?

No, I haven't, I badly needed to import some EML files for my work and I quickly downgraded to v.4.0.3 which works beautifully. In v.4.1.0, nothing happened, neither exporting nor importing.

> did you also observe the change in menu position?

Nope! Both kept the same position at the end of the Tool-menu ("Outils" in the French version of Thunderbird).

I hope it helps.

**Comment by picrochole** [comment]
@cleidigh 

I wanted to be 100% sure about the menu position; I then deleted v.4.0.3 to install v.4.1.0 again. Believe it or not, everything now works as expected, importing, exporting files, zipping folders, etc.

One possible explaination could be that the deletion of previous extension instead of its updating played some role (maybe by removing some conflicting element?)

**Comment by cleidigh** [comment]
@picrochole 
Excellent!  Potential problems . I cannot reproduce can take a long time to resolve, I'm glad it was just a reinstallation issue.  I believe your suspicion is probably correct.  Hopefully you can now try the new features , including fully localized help (more content still needed, but now I have the infrastructure)  any feedback would be great!
@Foxfan100 
Can you do a complete uninstall, reboot twice, reinstall?
@cleidigh
